Brokerages/Oanda/DataType/Account.cs

```diff
@@ -0,0 +1,161 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+﻿namespace QuantConnect.Brokerages.Oanda.DataType
+{
+    /// <summary>
+    /// Represents the Oanda Account.
+    /// </summary>
+    public class Account
```

Comment:
I know some may think this is nitpicky, but let's please use 4 spaces instead of tabs. If it's not already in CONTRIBUTING.md then I'll add it there now.

Was this code copied from somewhere? If it was we can leave it how it is but put a <remark /> at the top of the class with a link and/or description of the source.

Sometimes it's good to leave it exactly as copied so its easy to diff later and merge in bug fixes.